app/(main)/manager/layout.tsx

```
@@ -0,0 +1,95 @@
+    }, [isAuthorized, router]);
+
+    if (isAuthorized === null) {
+        return null;
+    }
+
+    if (!isAuthorized) {
+        return null;
+    }
+
+    return (
+        <html>
```

Biome (1.9.4)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)